.github/workflows/scheduled.yml

```diff
@@ -293,6 +293,7 @@ jobs:
                 --seed ${RANDOM} \
                 --enable_variadic_signatures \
                 --velox_fuzzer_enable_complex_types \
+                --velox_fuzzer_enable_decimal_type \
```

Comment:
It might be better to enable this on our experimental runs first and get a few green runs before we enable it for our nightly and CI  jobs.

Reply:
Got it. Thanks.